components/ActionsDropdown/styles.ts

```diff
@@ -90,3 +90,22 @@ export const StyledActionsDropdownArrow = styled(Box)<IBoxComponentProps>`
 `;
 
 StyledActionsDropdownArrow.displayName = 'StyledActionsDropdownArrow';
+
+
+export const StyledActionsDropdownButtonClose = styled(Box)<IBoxComponentProps>`
+	width: auto;
+	height: 56px;
+	border-radius: 14px;
```

Comment:
keep border radius the same as the list of options, feel free to chose the one from figma and apply it to both